main.go

```diff
@@ -70,6 +100,10 @@ Directive format and the powerful Runnable API using a variety of languages.`,
 	cmd.SetVersionTemplate("{{.Version}}\n")
 
 	cmd.Flags().Bool(waitFlag, false, "if passed, Atmo will wait until a bundle becomes available on disk, checking once per second")
+	cmd.Flags().String(appNameFlag, "Atmo", "if passed, it'll be used as ATMO_APP_NAME, otherwise 'Atmo' will be used")
+	cmd.Flags().String(domainFlage, "", "if passed, it'll be used as ATMO_DOMAIN and HTTPS will be used, otherwise HTTP will be used")
```

Comment:
```suggestion
	cmd.Flags().String(domainFlag, "", "if passed, it'll be used as ATMO_DOMAIN and HTTPS will be used, otherwise HTTP will be used")
```